tests/unittests/OnnxImporterTest.cpp

```diff
@@ -172,6 +172,89 @@ Error checkConstFoldedOutput(std::string NetFilename,
   return Error::success();
 }
 
+// Import reduceL2
```

Comment:
nit: Improve comment?

Reply:
I will remove it since it is unnecessary (following "testImportPRelu" case)